### PR TITLE
queryTransformer no longer supported

### PR DESCRIPTION
**Pull request by kishorevarma**
it would be misguiding new users

### CODE REVIEW

File: source/cache-updates.md [hunk 1]
```diff
@@ -41,16 +41,16 @@ const client = new ApolloClient({
...
 ```
-import ApolloClient, {createNetworkInterface, addTypename} from 'apollo-client'
+import ApolloClient, {createNetworkInterface} from 'apollo-client'
 
 const networkInterface = createNetworkInterface('http://localhost:3000/graphql') // TBD: Need to provide the right path for production
 
 const apolloClient = new ApolloClient({
     networkInterface: networkInterface,
-    queryTransformer: addTypename,
+    addTypename: true,
```
Comment (stubailo) [inline source/cache-updates.md]: This is on by default - so perhaps we can simply remove this section, or explain why someone might want to set `addTypename: false`.

Comment (kishorevarma) [inline source/cache-updates.md]: yeah that would be good,  and its explained why we need to use , so I don't think we need to explain why we should not use, its my opinion 

> // If the IDs are only unique per type (this is typical if an ID is just an
// ID out of a database table), you can use the `__typename` field to scope it.
// This is a GraphQL field that's automatically available, but you do need
// to query for it also

**Comment by stubailo** [comment]
If you have time please update the PR!

File: source/cache-updates.md (hunk 1 above)
Comment (kishorevarma) [inline source/cache-updates.md]: I will modify it, and remove the section.

**Comment by helfer** [comment]
Sorry, i didn't see this PR and just removed the offending lines today.

**Pull request closed without merging**